docs/content/python/guides/roles/getting-started.md

```diff
@@ -0,0 +1,273 @@
+application, you may attach this instance to the global flask object, or the
+current application if it needs to be accessed outside of the application
+initialization process.
+
+### Loading our policy
+
+Oso uses the Polar language to define authorization policies. An
+authorization policy specifies what requests are allowed and what data a
+user can access. The policy is stored in a Polar file, along with your
+code.
+
+Load the policy with the `Oso.load_file` function.
+
+```py
+oso.load_file("authorization.polar")
+```
+
+### Enable Oso roles
+
+
+In order to enable the built-in roles features, we 
+`oso.Oso.enable_roles` method:
+
+```py
+oso.enable_roles()
+```
+
+{{% callout "Load policies before enabling roles" "blue" %}}
+
+Oso will validate your roles configuration when you call `enable_roles`.
+You must load all policy files before enabling roles.
+
+{{% /callout %}}
+
+
+## Controlling access with roles
+
+Now, let's write our first rules that use role based access control. To
+setup the role library, we must:
+
+1. Add role and resource configurations to our policy.
+2. Use the `role_allows` method in our policy.
+3. Assign roles to users.
+
+### Configuring our first resource
+
+Roles in Oso are scoped to resources. A role is a grouping of
+permissions that may be performed on that resource. Roles are assigned
+to actors to grant them all the permissions the role has.
+
+We define resources in Polar using the `resource` rule. The `Org`
+resource represents an Organization in the GitClub example application.
+Let's walk through the resource definition for `Org`.
+
+```polar
+resource(_type: Org, "org", actions, roles) if
+```
+
+The rule head has 4 parameters:
+
+- `_type` is the Python class the resource definition is associated with.
+- `"org"` is the identifier for this resource type (this can be any string
+  you choose).
+- `actions` are set in the rule body. The `actions` list defines all the
+  actions that may be performed on the resource.
+- `roles` are set in the rule body. The `roles` dictionary defines all the
+  roles for this resource.
+
+In our rule body, we first define the list of available actions for this
+resource:
+
+```polar
+resource(_type: Org, "org", actions, roles) if
+    actions = ["read", "create_repo"] and
+    roles = {
+        ...
+    };
+```
+
+Now, we define our roles. Roles are defined in a dictionary that maps
+the role name to a role configuration.
+
+```polar
+resource(_type: Org, "org", actions, roles) if
+    actions = ["read", "create_repo"] and
+    roles = {
+        member: {
+            permissions: ["read"],
+        },
+        owner: {
+            permissions: ["read", "create_repo"],
+        }
+    };
+```
+
+This resource definition defines two roles:
+
+- _member_: Has the `read` permission.
+- _owner_: Has the `read` and `create_repo` permissions.
+
+Permissions are actions associated with a resource type. A permission can
```

Comment:
This is a good definition — maybe moving it up would address my previous comment about 'permission' vs. 'action'

Can be a follow-up

Reply:
Yep, follow up.